src/simulation/LoadGenerator.cpp

```diff
@@ -2124,20 +2240,52 @@ GeneratedLoadConfig::getSorobanUpgradeConfig() const
     return sorobanUpgradeConfig;
 }
 
+GeneratedLoadConfig::MixClassicSorobanConfig&
+GeneratedLoadConfig::getMutMixClassicSorobanConfig()
+{
+    releaseAssert(mode == LoadGenMode::MIXED_CLASSIC_SOROBAN);
+    return mixClassicSorobanConfig;
+}
+
+GeneratedLoadConfig::MixClassicSorobanConfig const&
+GeneratedLoadConfig::getMixClassicSorobanConfig() const
+{
+    releaseAssert(mode == LoadGenMode::MIXED_CLASSIC_SOROBAN);
+    return mixClassicSorobanConfig;
+}
+
 uint32_t&
 GeneratedLoadConfig::getMutDexTxPercent()
 {
-    releaseAssert(mode == LoadGenMode::MIXED_TXS);
+    releaseAssert(mode == LoadGenMode::MIXED_CLASSIC);
     return dexTxPercent;
 }
 
 uint32_t const&
 GeneratedLoadConfig::getDexTxPercent() const
 {
-    releaseAssert(mode == LoadGenMode::MIXED_TXS);
+    releaseAssert(mode == LoadGenMode::MIXED_CLASSIC);
     return dexTxPercent;
 }
 
+uint32_t
+GeneratedLoadConfig::getMinSorobanPercentSuccess() const
+{
+    releaseAssert(isSoroban());
+    return minSorobanPercentSuccess;
```

Comment:
nitpick: member var prefix `mMinSorobanPercentSuccess`

Reply:
Fixed in afcaba136d80585906c569af967c3372e3876ce3